assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -209,7 +209,11 @@ const MediaElement = ({
           setShowVideoDetail(false);
           if (mediaElement.current) {
             // Pointer still in the media element, continue the video.
-            mediaElement.current.play().catch(() => {});
+            const playPromise = mediaElement.current.play();
+            if (playPromise) {
```

Comment:
Reverted change from https://github.com/google/web-stories-wp/pull/2286/commits/efb575a3d5bbaf107f59ae6ccc4b826915d62141

I think the unit test runner doesn't return a promise so the test fails here.